modules/data/spec/entity-services/entity-collection-service.spec.ts

```diff
@@ -133,9 +134,16 @@ describe('EntityCollectionService', () => {
       const httpError = { error: new Error('Test Failure'), status: 501 };
       const error = makeDataServiceError('GET', httpError);
       dataService.setErrorResponse('getWithQuery', error);
-      heroCollectionService
-        .getWithQuery({ name: 'foo' })
-        .subscribe(expectErrorToBe(error, done));
+      heroCollectionService.getWithQuery({ name: 'foo' }).subscribe(
+        () => {
+          console.log('expected error', error);
+          expectErrorToBe(error, done);
+        },
+        (err) => {
+          console.log('actual error', err);
+          expectErrorToBe(error, done);
+        }
+      );
```

Comment:
Only this test is currently failing. To make it work, we need to revert this change